src/components/Homepage/Homepage.js

```diff
@@ -3,6 +3,11 @@ import * as styles from '../../styles/HomePage.module.css';
 import { Nav } from './Nav/Nav.js';
 
 function HomePage(props) {
+  if (localStorage.getItem('userInput') === null) {
+    const userInput = {};
+    localStorage.setItem('userInput', JSON.stringify(userInput));
+  }
+
```

Comment:
I can't help but wonder why do you have this here

Reply:
you can just create it when/if you need it. We put the auth logic when the app loads because you can't use the app without it. This is not the case here.